lms/routes.py

```diff
@@ -129,6 +129,11 @@ def includeme(config):  # pylint:disable=too-many-statements
         "admin.instance.new.registration", "/admin/instance/new/registration"
     )
 
+    org_public_id_pat = r"\w{2}\.lms\.org\.\w+"
+    config.add_route(
```

Comment:
Using the public_id on the URL.

In application instance we use either ID or consumer key that's a bit of an accident.

Reply:
Should we be this picky about the format? If we change it this would break. Are you imagining we'll arrive here with different ids and need to route?

Reply:
I don't think this regex is right as it doesn't allow dashes as in: `18RYrpLBR1egA-NE4f7A_g`. I'd probably be in favour of dropping it.

Reply:
I'm coming to the conclusion we might be making things overly difficult by using the public id here:

 * Currently this is the only way to access the page, but it won't be soon
 * Keying by primary key would avoid any complication around parsing the id